src/client/containers/ProductPage/ProductPage.Container.js

```diff
@@ -1,12 +1,87 @@
+/* eslint-disable no-console */
+/* eslint-disable radix */
```

Comment:
You should not disable eslint you should address first and try to fix those errors.
You can easily fix this. The onnly eeror you need to disable is `camelcase `- becase thats coming from DB

Reply:
done Julia..